models/concerns/validate_data.rb

```diff
@@ -298,10 +345,33 @@ def check_relations(periodic_heuristic)
     )
   end
 
+  def check_sticky_relation_consistency
+    unconsistent_stickies = []
+    @hash[:relations].none?{ |relation|
```

Comment:
If I am not mistaken in the VRP hash any key can be `nil`, right? Either we should initialise `hash[:relations] ||= []` as we do for vehicles and services at the beginning of `check_consistency` or we need to do as follows...
```suggestion
    @hash[:relations]&.none?{ |relation|
```

I prefer the first solution (i.e., initialisation), in that case it is better to clean the unnecessary `&.`, what do you think @braktar ?